src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.ts

```diff
@@ -7,19 +7,19 @@ import { Component, Input, OnChanges } from '@angular/core';
 })
 export class UsageBarComponent implements OnChanges {
   @Input()
-  totalBytes: number;
+  total: number;
   @Input()
-  usedBytes: number;
+  used: number;
+  @Input()
+  isBinary: boolean;
```

Comment:
`isBinary` should be `true` by default unless you want to use the `dimless` pipe as new default everywhere ;)
```suggestion
  isBinary = true;
```

Reply:
Done!